Modularize, update, and enhance user focus of the "Architecture" part of Planning_for_Project (#2607)
theforeman/foreman-documentation: guides/common/modules/con_smartproxy-overview.adoc

[id="{SmartProxy}-Overview_{context}"]
= {SmartProxy} overview

With {SmartProxyServers}, you can extend the reach and scalability of your {Project} deployment.
{SmartProxyServers} provide the following functionalities in a {ProjectName} deployment:

> * Mirroring content from {ProjectServer} to establish content sources in various geographical locations.

Reviewer: ```suggestion
* Mirroring content from {ProjectServer} to establish content sources in various geographical and logical locations.
```

maybe "logical" for multiple networks (DMZs) in the same rack/data center?

Author: Applied.

> By assigning a host system to a {SmartProxyServer}, you can configure this host to pull content and configuration from the {SmartProxy} in their location instead of from the central {ProjectServer}.

Reviewer: pull > receive?

Author: Applied.

> By assigning a host system to a {SmartProxyServer}, you can configure this host to receive content and configuration from the {SmartProxy} in their location instead of from the central {ProjectServer}.

Reviewer: ```suggestion
By registering a host to a {SmartProxyServer}, you can configure this host to receive content and configuration from the {SmartProxy} in their location instead of from the central {ProjectServer}.
```

* Running localized services to discover, provision, control, and configure hosts.

> You can specify the exact subset of content that {SmartProxyServer} makes available to hosts by using content views.

Reviewer: Maybe reword to "You can use content views to specify the exact subset of content that {SmartProxyServers} provide to hosts"?

Author: Applied.

[id="{SmartProxy}-Overview_{context}"]
= {SmartProxy} Overview

> With {SmartProxyServers}, you can extend the reach and scalability of your {Project} deployment. {SmartProxyServers} provide the following functionalities in a {ProjectName} deployment:

Reviewer 2: ```suggestion
With {SmartProxyServers}, you can extend the reach and scalability of your {Project} deployment.
{SmartProxyServers} provide the following functionalities in a {ProjectName} deployment:
```
Recalling that meme you posted :smile:
